pkg/kubelet/allocation/state/state_mem.go

```diff
@@ -59,6 +59,23 @@ func (s *stateMemory) GetContainerResources(podUID types.UID, containerName stri
 	return *resources.DeepCopy(), ok
 }
 
+// GetPodLevelResources returns current resources information at pod-level
+func (s *stateMemory) GetPodLevelResources(podUID types.UID) (*v1.ResourceRequirements, bool) {
+	s.RLock()
+	defer s.RUnlock()
+
+	pr, ok := s.podResources[podUID]
+	if !ok {
+		return nil, ok
+	}
+
+	if pr.PodLevelResources != nil {
```

Comment:
Shouldn't this be `if pr.PodLevelResources == nil`? This seems like a pretty fundamental problem - unless I'm missing something, it seems concerning that tests are passing with this error.